lib/Space.js

```diff
@@ -243,7 +243,7 @@
         space[key] = new Space(val, key, space, prevSpace);
       } else {
         Object.defineProperty(space, key, {
-          value: Object.freeze(val),
+          value: val,
```

Comment:
`Object.freeze` is supported by IE9+.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/freeze

Reply:
Can you tell me more about the problem you're encountering?

Reply:
`Object.freeze` works in IE9+, however IE11 or lower will throw unless specifically given an object. The API specifically states it Freezes **OBJECTS** not strings, numbers, etc. which would be pointless since strings, numbers, etc are all equal regardless. So this call effectively does nothing.

Reply:
@JonAbrams can we please get this or something working in IE11 merged.

Reply:
Thanks for the explanation!

I'll merge this in and publish a new version tonight.